src/workerd/api/http-test.js

```diff
@@ -206,44 +206,56 @@ export const inspect = {
   }
 };
 
+async function requestCacheThrowsError(cacheHeader, errorName, errorMessage){
```

Comment:
I suggest naming them all `assert*` since these are, essentially assertions.

Or `verify*` if you think they will be complicated.